index.html

```diff
@@ -1,177 +1,238 @@
-            <ul class="social-icons">
-                <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
-                <li><a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
-                <li><a href="#"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
-                <li><a href="#"><i class="fa fa-github" aria-hidden="true"></i></a></li>
-                <li><a href="#"><i class="fa fa-angellist" aria-hidden="true"></i></a></li>
-            </ul>
-            <button type="button" class="project-btn">Get My Resume</button>
+          <h2 class="icon-text">LET'S CONNECT</h2>
+          <ul class="social-icons">
+            <li>
+              <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-medium" aria-hidden="true"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-github" aria-hidden="true"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-angellist" aria-hidden="true"></i></a>
+            </li>
+          </ul>
+          <button type="button" class="about-btn">Get My Resume</button>
         </div>
-        <div class="languages-down">
-          <h3>Languages</h3>
-          <i class="fa fa-angle-down" aria-hidden="true"></i>
-        </div>
-        <div class="dropdown-technologies">
-          <img src="./Images/javascript.png" alt="javaScript">
-          <p>Javascript</p>
-        </div>
-        <div class="dropdown-technologies">
-          <img src="./Images/HTML.png" alt="html">
-          <p>HTML</p>
-        </div>
-        <div class="dropdown-technologies">
-          <img src="./Images/CSS.png" alt="css">
-          <p>CSS</p>
-        </div>
-        <div class="languages-down">
-          <h3 class="space-down">Frameworks</h3>
-          <i class="fa fa-angle-right" aria-hidden="true"></i>
-        </div><hr>
-        <br>
-        <div class="languages-down">
-          <h3 class="space-down">Skills</h3>
-          <i class="fa fa-angle-right" aria-hidden="true"></i>
-        </div><hr>  
-    </section>
+        <ul class="tech-space">
+          <li class="languages-down tech-space">
+            <h3>Languages</h3>
+            <i class="fa fa-angle-down" aria-hidden="true"></i>
+          </li>
+          <li class="dropdown-technologies">
+            <img src="./Images/javascript.png" alt="javaScript" />
+            <p>Javascript</p>
+          </li>
+          <li class="dropdown-technologies">
+            <img src="./Images/HTML.png" alt="html" />
+            <p>HTML</p>
+          </li>
+          <li class="dropdown-technologies">
+            <img src="./Images/CSS.png" alt="css" />
+            <p>CSS</p>
+          </li>
```

Comment:
- [ ] [OPTIONAL] The selected lists have been given a border-radius of 7% as seen below
```
.dropdown-technologies {
    background-color: #f7f7f9;
    display: flex;
    padding: 7px;
    align-items: center;
    margin: 10px 0;
    border-radius: 7%; 👈🏻 
}
```
This causes an inconsistent shape as seen below

| Your design | Figma design |
| --- | --- |
| <img width="303" alt="shap" src="https://user-images.githubusercontent.com/4247740/143687431-1dd21458-92a3-472e-ab2d-814619318b09.PNG"> | <img width="246" alt="shap2" src="https://user-images.githubusercontent.com/4247740/143687436-320aadfc-6565-4ba3-955c-3aa61a02621e.PNG"> |

I there fore suggest that you change the "%" units to "px" to correct the design.

Reply:
- Done.